cffi.asd

```diff
@@ -65,11 +65,28 @@
      (:file "foreign-vars")
      (:file "features")))))
 
-(defmethod operation-done-p ((o test-op) (c (eql (find-system :cffi))))
-  nil)
-
-(defmethod perform ((o test-op) (c (eql (find-system :cffi))))
-  (operate 'asdf:load-op :cffi-tests)
-  (operate 'asdf:test-op :cffi-tests))
```

Comment:
This makes `(asdf:test-system :cffi)` not do anything. It's not such a great loss, but what was the goal here?